packages/core/helper-plugin/src/utils/request.ts

```diff
@@ -35,44 +36,36 @@ function checkStatus(response, checkToken = true) {
 
   return parseJSON(response)
     .then((responseFormatted) => {
-      const error = new Error(response.statusText);
+      const error: CustomError = new Error(response.statusText);
       error.response = response;
       error.response.payload = responseFormatted;
       throw error;
     })
     .catch(() => {
-      const error = new Error(response.statusText);
+      const error: CustomError = new Error(response.statusText);
       error.response = response;
 
       throw error;
```

Comment:
I don't think you need to change this because the file is deprecated anyway, but in case you werent aware, you can always make a new Error by doing:

```ts
class ResponseError extends Error {
	constructor(response: Response) {
		this.response = response
		this.response.payload = responseFormatted

		super(response.statusText)
	}
}
```

Which would be better in this case overall imo. But I wouldn't make the change 😄